internal/resources/resource_registry_secret.go

```diff
@@ -72,9 +73,19 @@ func (r *registrySecretResource) Schema(_ context.Context, _ resource.SchemaRequ
 				Description: "The URL of the container registry (e.g., ghcr.io)",
 			},
 
-			"auth_token": schema.StringAttribute{
-				Required:    true,
-				Description: " authentication token for the registry",
+			"plaintext_value": schema.StringAttribute{
+				Optional:    true,
+				Description: "plain text auth token for the registry",
+				Validators: []validator.String{
+					SecretValidator(Registry),
+				},
+			},
+			"encrypted_auth_token": schema.StringAttribute{
```

Comment:
We should keep this consistent too, where `encrypted_auth_token` becomes `encrypted_value`  to match the syntax of `plaintext_value`